membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -34,12 +33,6 @@ class TouchpointComponents(stage: String)(implicit  system: ActorSystem, executi
   lazy val conf = Config.config.getConfig("touchpoint.backend")
   lazy val environmentConf = conf.getConfig(s"environments.$stage")
 
-  lazy val identityConfig = new IdentityConfig {
-    override def token: String = environmentConf.getString("identity.marketingToken")
-    override def url: String = environmentConf.getString("identity.apiUrl")
-  }
-
-  lazy val identityService = new IdentityService(identityConfig, RequestRunners.futureRunner)
   lazy val digitalPackConf = environmentConf.getConfig(s"zuora.ratePlanIds.digitalpack")
   lazy val paperCatalogConf = environmentConf.getConfig(s"zuora.productIds.subscriptions")
   lazy val membershipConf = environmentConf.getConfig(s"zuora.ratePlanIds.membership")
```

Comment:
Can dynamoBehaviourTable be removed here?

Reply:
Yep, good catch - thanks